routes/index.js

```diff
@@ -23,21 +23,21 @@ router.get('/overlays/scores', function(req, res) {
 
 /* GET home page. */
 router.get('/overlays/grimetime/game4by3', function(req, res) {
-  res.render('overlays/grimetime/game4by3', { title: 'Overlay', layout: false });
+  res.render('overlays/grimetime/game4by3', { title: 'Overlay' });
 });
 
 /* GET home page. */
 router.get('/overlays/grimetime/game16by9', function(req, res) {
-  res.render('overlays/grimetime/game16by9', { title: 'Overlay', layout: false });
+  res.render('overlays/grimetime/game16by9', { title: 'Overlay' });
 });
 
 /*UMSK*/
 router.get('/overlays/umsk/game16by9', function(req, res) {
-  res.render('overlays/umsk/game16by9', { title: 'UMSK Overlay', layout: false });
+  res.render('overlays/umsk/game16by9', { title: 'UMSK Overlay'});
 });
 
 router.get('/overlays/umsk/game4by3', function(req, res) {
-  res.render('overlays/umsk/game4by3', { title: 'UMSK Overlay', layout: false });
+  res.render('overlays/umsk/game4by3', { title: 'UMSK Overlay' });
```

Comment:
Using the layout here is going to make the overlay pages pretty jank, because they are already full HTML pages with a `<head>` and such. I think it would be useful to have a base layout for the overlays, but we'd probably want to make a separate one and refactor all of the overlays to be based off of it.

For now, I think it'd be best to set layout back to false, then add the character stylesheet to each of the overlays.